exercises/concept/elyses-transformative-enchantments/.docs/introduction.md

```diff
@@ -1,23 +1,181 @@
 # Introduction
 
-In JavaScript, the `Array` class has many powerful built-in functions for transforming arrays. These functions make it much easier to do things than it otherwise would be using a simple for loop or more direct manipulation.
+In JavaScript, the `Array` class has many powerful built-in methods for transforming arrays.
+These methods make it much easier to convert one array into another than it otherwise would be using a simple for loop or more direct manipulation.
 
-Example of transformation using a for loop and second array:
+Some methods are [_pure_][pure-function-definition], meaning that they no not modify the original array.
+They return a new one instead.
+Other methods however manipulate the array they are called on and not **not** return the modified array.
+
+Here is a presentation of some of the most commonly used methods.
```

Comment:
Should there be a link to the presentation

Reply:
What is meant here is that the methods are presented below. I rephrased the sentence to make this more clear.